### PR TITLE
Move to using fixed sizes on all buttons in line with Apple's recommendation

### PR DESCRIPTION
**Pull request by finben**

### CODE REVIEW

**Comment by finben** [comment]
Simulator Screen Shot - iPhone 8 - 2020-06-18 at 13 13 51
Simulator Screen Shot - iPhone 8 - 2020-06-18 at 13 10 41

**Comment by finben** [comment]
@skywalkerdude  We may need to do the alternate layout she mentions at minute 15 of https://developer.apple.com/videos/play/wwdc2017/245/ for tag labels. To put them in a VStack instead of HStack when the user scales up dynamic type to high

File: Hymns/Home/HomeContainerView.swift [hunk 1]
```diff
@@ -9,22 +9,24 @@ struct HomeContainerView: View {
...
-                    .tabItem {HomeTab.favorites.getImage(selectedTab == HomeTab.favorites).imageScale(.large)}
+                    .tabItem {HomeTab.favorites.getImage(selectedTab == HomeTab.favorites).font(.system(size: 25))
+}
                     .tag(HomeTab.favorites)
                     .hideNavigationBar()
 
                 SettingsView()
-                    .tabItem {HomeTab.settings.getImage(selectedTab == HomeTab.settings).imageScale(.large)}
+                    .tabItem {HomeTab.settings.getImage(selectedTab == HomeTab.settings).font(.system(size: 25))
+}
```
[the remark this answers is not in the document]
Comment (finben) [reply Hymns/Home/HomeContainerView.swift]: addressed

File: Hymns/Home/HomeContainerView.swift [hunk 2]
```diff
@@ -9,22 +9,24 @@ struct HomeContainerView: View {
...
                 BrowseView()
-                    .tabItem { HomeTab.browse.getImage(selectedTab == HomeTab.browse).imageScale(.large)}
+                    .tabItem { HomeTab.browse.getImage(selectedTab == HomeTab.browse).font(.system(size: 25))}
                     .tag(HomeTab.browse)
                     .hideNavigationBar()
 
                 FavoritesView()
-                    .tabItem {HomeTab.favorites.getImage(selectedTab == HomeTab.favorites).imageScale(.large)}
+                    .tabItem {HomeTab.favorites.getImage(selectedTab == HomeTab.favorites).font(.system(size: 25))
+}
```
[the remark this answers is not in the document]
Comment (finben) [reply Hymns/Home/HomeContainerView.swift]: addressed

**Comment by finben** [comment]
I moved alot more of them to the new public let buttonSize being 20. I removed the 25 one because I don't think we even need it. They should be able to mostly all be the same size except for the music player to make the play button stand out. So I created a largeButtonSize for that.

**Comment by skywalkerdude** [comment]
@finben I adjusted some sizes and added some scrollviews to make things look closer to the default when it's at a normal size. Could you do a quick sweep to make sure it all looks good on normal size and large size? Then go ahead and submit if all looks good to you!

**Comment by skywalkerdude** [comment]
close https://github.com/HymnalDreamTeam/Hymns-iOS/issues/314

**Comment by skywalkerdude** [comment]
close https://github.com/HymnalDreamTeam/Hymns-iOS/issues/324

**Comment by skywalkerdude** [comment]
close https://github.com/HymnalDreamTeam/Hymns-iOS/issues/316

**Comment by skywalkerdude** [comment]
close #309

**Comment by skywalkerdude** [comment]
close https://github.com/HymnalDreamTeam/Hymns-iOS/issues/323

**Comment by skywalkerdude** [comment]
@finben I think you need to update the snapshot tests for this to pass

**Comment by finben** [comment]
@skywalkerdude I tried to change the record settings like you said to true. But that didn't work. Is that what you do to reset snapshot tests?

**Comment by finben** [comment]
@skywalkerdude Idk why this isn't passing now. I re recorded all snapshots and they all pass on my local now.

**Comment by finben** [comment]
Did you commit all the changes you made? When I pulled in your changes I started getting the following error. 
error: Build input file cannot be found: '/Users/ben/dev/hymns3/Hymns-iOS/Hymns/Debug Implementations/TestHymnDataStore.swift' (in target 'Hymns' from project 'Hymns')

**Pull request closed without merging**